docs/elements.md

```diff
@@ -351,19 +351,50 @@ The contents of the file editor are only displayed by default in the question pa
 Provides a way to accept file uploads as part of an answer. They will be stored
 in the format expected by externally graded questions.
 
+Note that there is a file size limit of **5 MB per answer**. This limit is not customizable as larger
+requests will be rejected by the server. For the same reason, it is also not possible to bypass the
+limit by using multiple `pl-file-upload` elements in the same question. To avoid unexpected errors or
+potentially misleading error messages for large file uploads, we recommend not using more than one
+`pl-file-upload` element per question.
+
 #### Sample element
 
 ![Screenshot of the pl-file-upload element](elements/pl-file-upload.png)
 
 ```html title="question.html"
-<pl-file-upload file-names="foo.py, bar.c, filename with\, comma.txt"></pl-file-upload>
+<pl-file-upload
+  file-names="foo.py, bar.c, filename with\, comma.txt"
+  optional-file-names="foo.pdf, *.py, file_?.txt, file_[abc].txt, file with \\? as non-wildcard.pdf"
+></pl-file-upload>
```

Comment:
This no longer matches the screenshot below. I'd be inclined to revert this and rely just on the documentation below for the new attributes.